Add TF Object Detection CSV Format

phonymacoroni opened the pull request:
If you want to run locally it will not work until merged with main unless you comment out the building of containers and build them yourself. See commit `a343981981e7dc073f11e6f85cf53a107a0ba4c3`

Octogonapus commented on ui-vaadin/src/main/kotlin/edu/wpi/axon/ui/view/preferences/PluginEditorDialog.kt:
```diff
@@ -0,0 +1,81 @@
+import edu.wpi.axon.plugin.PluginManager
+import edu.wpi.axon.ui.view.HasNotifications
+
+class PluginEditorDialog(pluginManager: PluginManager, bean: Plugin? = null, onSave: (Plugin) -> Unit = {}) : KComposite(), HasNotifications {
```
Is this line too long? Probably need each parameter on a new line.

Octogonapus commented on ui-vaadin/src/main/kotlin/edu/wpi/axon/ui/view/preferences/PluginEditorDialog.kt:
```diff
@@ -0,0 +1,81 @@
+    init {
+        ui {
+            dialog {
+                dialog = this
```
Can't you do `dialog = dialog { ...` instead?

phonymacoroni committed:
Groundwork for accepting tensorflow object detection csv formats as zips

Instructions:
in 'containers/mobilenet/dataset' run 'docker build -t newdatareader:edge .'
Then use 'docker dev' or 'yarn start' to run, do not use 'yarn dockerdev' or pull from dockerhub.

phonymacoroni committed:
Groundwork for accepting tensorflow object detection csv formats as zips

Instructions:
in 'containers/mobilenet/dataset' run 'docker build -t newdatareader:edge .'
Then use 'docker dev' or 'yarn start' to run, do not use 'yarn dockerdev' or pull from dockerhub.

phonymacoroni committed:
Unzip files

map.pbtxt errors

print statements included

phonymacoroni committed:
Unzip files

map.pbtxt errors

print statements included

phonymacoroni committed:
File path corrected

Reading csvs and parse_meta changes

phonymacoroni committed:
File path corrected

Reading csvs and parse_meta changes

phonymacoroni committed:
Hardcoded working

phonymacoroni committed:
Hardcoded working

phonymacoroni committed:
Editing training container

fix hanging arg

phonymacoroni committed:
Editing training container

fix hanging arg

phonymacoroni committed:
fix wrong name of record

phonymacoroni committed:
fix wrong name of record

phonymacoroni committed:
With testing changes

phonymacoroni committed:
With testing changes

phonymacoroni committed:
Add comment

phonymacoroni committed:
Add comment

phonymacoroni committed:
Updated paths

phonymacoroni committed:
Updated paths

phonymacoroni committed:
Removed comments for docker container

phonymacoroni committed:
Removed comments for docker container

phonymacoroni committed:
Comment for duplicate extraction

phonymacoroni committed:
Comment for duplicate extraction

phonymacoroni committed:
Merge remote-tracking branch 'origin/straight-tf' into straight-tf

# Conflicts:
#	containers/mobilenet/test/scripts/test.py

phonymacoroni committed:
Remove prints

phonymacoroni committed:
Add Fixed record swap

NLZ432 reviewed (approved):
nice
